data/human/human missions.txt

```diff
@@ -6893,3 +6893,53 @@ mission "Returning to Paradise"
 			action
 				payment 120000
 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`
+
+mission "Hogshead's Dilemma"
+	minor
+	invisible
+	source
+		near Sol 1 10
+	to offer
+		random < 5
+	on offer
+		conversation
+			`While passing through the spaceport, you notice a few energetic pilots gathered around a computer terminal, invested in some sort of conversation. One of the pilots speaks louder than the rest, sometimes getting looks from others in the spaceport. "You've fooled me before, and I reckoned you were this time! And here I thought them dang things were old news!" The pilot lets out a hearty chuckle. "Even nowadays them Firkins are still getting names!" After another minute or two, the group walks away from the terminal and out of your sight.`
```

Comment:
I must be out of touch - what is a `Firkin`?

Reply:
~~9 gallons~~

Reply:
The Hogshead's name is a synonym of Keg, which the engines look like. Firkin is also a synonym for that. Both were options for the ship's name, but the former was chosen, so throwing the latter here as another "nickname" for it, and the ship's lore is all about nicknames.

Reply:
Ah, gotcha. Makes sense!